projects/plugins/jetpack/extensions/blocks/subscriptions/utils.js

```diff
@@ -69,3 +71,16 @@ export default function GetAddPaidPlanButton( { context = 'other', hasNewsletter
 		</Button>
 	);
 }
+
+export const GetAccessLevel = postType => {
```

Comment:
`getAccessLevel` since not a component.

Reply:
could be good use for a hook https://react.dev/learn/reusing-logic-with-custom-hooks